### DISCUSSION TITLE
Feature Request: Option to add "Modern Apps" into a folder for the Windows 10 Start Menu.

### DISCUSSION THREAD

**Discussion by SmilerRyan** [valinet/ExplorerPatcher#2414] [Ideas]
For example, most other replacements put the modern/metro/store apps into it's own folder, such as start11/classicshell/startallback etc.

Here's an example of my "Apps" folder in the start menu (In this picture, I use Windows 10 with StartIsBack):

![image](https://github.com/valinet/ExplorerPatcher/assets/14893385/672af108-408a-447f-89a8-90a67b24e444)

On Windows 11 with ExplorerPatcher, i'd like an option to hide the modern apps and have them in a new folder, like this:
![image](https://github.com/valinet/ExplorerPatcher/assets/14893385/87842961-e39d-4907-8b2a-3f9199525866)

(edit for better clarity, before and expected after with the start menu)

![old](https://github.com/valinet/ExplorerPatcher/assets/14893385/e31257ea-1289-4774-adac-b4c5ac704c33)

![new](https://github.com/valinet/ExplorerPatcher/assets/14893385/107e31f5-84f4-4858-a560-54d68d941282)

**Comment by pyrates999** [comment]
Just an FYI.  Classic shell is no longer maintained.  You should install open shell if you want to use it.

And this to me looks like it would be a feature for one of the start menu replacements, not EP.

**Comment by SmilerRyan** [comment]
Fair enough, I was just hoping to avoid installing multiple each time I reinstall windows, I actually prefer the windows 10 start menu over the others and the only reason I don't use it is because I can't hide/relocate the default apps.
